.phpcs.xml.dist

```diff
@@ -97,6 +103,11 @@
 		<exclude-pattern>/tests/bootstrap\.php</exclude-pattern>
 	</rule>
 
+	<rule ref="PHPCompatibility">
+	    <exclude name="PHPCompatibility.Classes.NewTypedProperties.Found"/>
```

Comment:
Please comment on why we're excluding these.

Reply:
All fixed :+1: